drivers/SmartThings/wemo/src/discovery.lua

```diff
@@ -49,7 +49,7 @@ end
 local function process_response(val)
   local info = {}
   val = string.gsub(val, "HTTP/1.1 200 OK\r\n", "", 1)
-  for k, v in string.gmatch(val, "([%g]+): ([%g ]*)\r\n") do
+  for k, v in string.gmatch(val, "([%w_-]+): *([%g ]*)\r\n") do
```

Comment:
nit: For zero or more spaces, this is typically written as the following in regex (`[ ]*`). I think it may be required in some languages but not lua.  At least for me, that threw me off at first.